users/src/entities/Activity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity, PrimaryKey, Property } from "@mikro-orm/core";
+
+@Entity({ tableName: "activity" })
+export class Activity {
+    @PrimaryKey()
+    uid!: string;
+
+    @PrimaryKey()
+    questionId!: number;
```

Comment:
minor nit / question than an actl comment: you're using `!:` in your code, does that do anything if strictNullChecks (in ur tsconfig.json) is false? I thought this is only relevant if strictNullChecks are on and you're telling tsc this value never needs to be checked for null/undefined.

ps its been a while since I used typescript, might be mistaken on how the definite assertion works.

Reply:
In the orm docs, they did like this for entity classes. So I just followed them.